SoftLayer/CLI/hardware/detail.py

```diff
@@ -71,9 +72,11 @@ def cli(env, identifier, passwords, price, components):
     table.add_row(['last_transaction', last_transaction])
     table.add_row(['billing', 'Hourly' if result['hourlyBillingFlag'] else 'Monthly'])
 
-    vlan_table = formatting.Table(['type', 'number', 'id'])
+    vlan_table = formatting.Table(['type', 'number', 'id', 'Name', 'Netmask'])
```

Comment:
Change the first letter of 'type', 'number' and 'id' to uppercase or the first letter of 'Name', 'Netmask' to lower case